openpype/modules/clockify/clockify_api.py

```diff
@@ -6,34 +6,21 @@
 import requests
 from .constants import (
     CLOCKIFY_ENDPOINT,
-    ADMIN_PERMISSION_NAMES
+    ADMIN_PERMISSION_NAMES,
+    MAX_CALLS,
+    PERIOD,
 )
 
 from openpype.lib.local_settings import OpenPypeSecureRegistry
 
 
-def time_check(obj):
-    if obj.request_counter < 10:
-        obj.request_counter += 1
-        return
-
-    wait_time = 1 - (time.time() - obj.request_time)
-    if wait_time > 0:
-        time.sleep(wait_time)
-
-    obj.request_time = time.time()
-    obj.request_counter = 0
-
-
 class ClockifyAPI:
     def __init__(self, api_key=None, master_parent=None):
         self.workspace_name = None
-        self.workspace_id = None
         self.master_parent = master_parent
         self.api_key = api_key
-        self.request_counter = 0
-        self.request_time = time.time()
-
+        self.__workspace_id = None
```

Comment:
Please don't use double underscore. It's potentially dangerous as python interpreter resolves then in a different way than other attributes.